### DISCUSSION TITLE
The sample in documentation for iOS does not exists

### DISCUSSION THREAD

**Discussion by hulkdx** [arkivanov/Decompose#740] [General]
Hi, when I click on
[compose-darwin/sample](https://github.com/arkivanov/Decompose/tree/compose-experimental/sample) 

Inside  https://arkivanov.github.io/Decompose/extensions/compose/#samples-for-compose-for-ios-and-web-jscanvas-not-wasm
it says 404 - page not found.

**Comment by arkivanov** [comment]
Thanks! That's section is quite old and is no longer relevant, I've removed it. All the samples are now in the main branch: https://github.com/arkivanov/Decompose/tree/master/sample.
